### PR TITLE
Fix identity case sensitivity loading in secondary cluster

### PR DESCRIPTION
**Pull request by vishalnayak**

### CODE REVIEW

File: vault/identity_store.go [hunk 1]
```diff
@@ -120,6 +133,38 @@ func (i *IdentityStore) Invalidate(ctx context.Context, key string) {
 	defer i.lock.Unlock()
 
 	switch {
+	case key == caseSensitivityKey:
+		entry, err := i.view.Get(ctx, caseSensitivityKey)
```
Comment (jefferai) [inline vault/identity_store.go]: There should be a nil check here.

**Comment by jefferai** [comment]
Could the primary switch modes after the initialization function is run?

**Comment by vishalnayak** [comment]
The mode is never flipped after the initialization function is run. The mode is only set during startup.

**Comment by jefferai** [comment]
I think it's looking pretty good...would like to see tests showing the problem and that this fixes it.

**Comment by vishalnayak** [comment]
I'll write a test post merge.

**Pull request merged**